generate/docker.sh

```diff
@@ -32,8 +32,6 @@ docker run --rm -it \
     -v $OPENAPI_PATH:/openapi.json \
     -v "$GEN_PATH":/gen \
     -v $(pwd)/generate.sh:/generate.sh \
-    -v $(pwd)/templates:/templates \
```

Comment:
@justinkoke Do you know we generate an empty templates folder and pass it to the generator?
It appears we do not have any templates so this seems unnecessary.

Reply:
> @justinkoke Do you know we generate an empty templates folder and pass it to the generator? It appears we do not have any templates so this seems unnecessary.

This is probably just a copy and paste @helenamariano so can just be removed.